url.bs

```diff
@@ -1120,15 +1120,13 @@ resource the <a for=/>URL</a>'s other components identify. It is initially null.
 <p id=non-relative-flag>A <a for=/>URL</a> also has an associated
 <dfn export for=url>cannot-be-a-base-URL flag</dfn>. It is initially unset.
 
-<p>A <a for=/>URL</a> also has an associated <dfn export for=url id=concept-url-object>object</dfn>
-that is null, a {{Blob}} object, or a {{MediaSource}} object. It is initially null.
-[[!FILEAPI]]
-[[!MEDIA-SOURCE]]
-
-<p class="note no-backref">At this point this is used primarily to support
-"<code>blob</code>" <a for=/>URLs</a>, but others can be added going forward, hence
-"object".
+<p>A <a for=/>URL</a> also has an associated <dfn export for=url id=concept-url-blob-entry>blob entry</dfn>
+that is either null or a <a>blob URL entry</a>. It is initially null.
```

Comment:
This seems reasonable. Any reason for the name difference?

Reply:
With the name change it's obvious that references to the old term need to be updated somehow. If I had just kept the old term it would be much harder to figure out where references to the old definition exist that now need to be updated because it is a different type. Also it seemed weird to be talking about a URL's object's object, but that's pretty minor of course.

Reply:
Sorry, I meant between "blob entry" and "blob URL entry".

Reply:
Ah, no, no particular reason. Changed it to "blob URL entry".